### DISCUSSION TITLE
Uploader bug issue

### DISCUSSION THREAD

**Discussion by hypertextPreprocessor** [rsuite/rsuite#1572] [General]
Uploader component can not do a same picture submit after a clean(onRemove handler),while with an action attribute on this component can fix this issue.but sometimes I don't need it auto upload for me.

**Comment by simonguo** [comment]
What version of rsuite are you using?

**Reply by hypertextPreprocessor** [reply]
my version is 4.8.4  and react  "^17.0.1","react-dom": "^17.0.1".
  (from package.json)

**Reply by simonguo** [reply]
https://github.com/rsuite/rsuite/blob/v4/CHANGELOG.md#493-2021-03-11
This issue has been fixed in `rsuite@4.9.3`.
